app/views/pages/home.html.erb

```diff
@@ -1,73 +1,87 @@
-    <% else  %>
-    <%= link_to "CONNECT", new_user_session_path, class:"hvr-sweep-to-right btn-orange"%>
-    <% end %>
+  <div class="home-top">
   </div>
-</div>
-<div class="home-wrapper">
-  <div class="home-title">About</div>
-  <div class="home-about">
-    <p> We all are blessed with a lot of <span class="home-highlight">skills</span>. Why not connect with the people around us by exchanging them? With RefuGapp we are filling the gap between locals and newcomers by providing a platform where you can offer your offer and find skill and <span class="home-highlight">connect</span> with new people! </p>
-    <p>Are you a <span class="home-highlight">local</span>? Get to now the newcomers in your city by exchanging skills! Discover how refugees can help you, send a request and.. connect! Or: a skill you want to share? Make a post and someone will find you!</p>
-    <p>Are you a <span class="home-highlight">newcomer</span>? Get to know the locals by offering your skills! Make a post and help a local with the things you are good at.
-      Or: in need of some help? Look what the locals are offering and send a request!</p>
+  <div class="home-middle">
+    <div class="home-logo"><%=image_tag "refuGapp.png", width: '300px'  %></div> <br>
+    <div class="home-tagline">Hallo - Hi - مرحبا </div>
+    <div class="home-button">
+      <% if current_user  %>
+      <%= link_to "What's new?", posts_path, class:"hvr-sweep-to-right btn-orange"%>
+      <% else  %>
+      <%= link_to "CONNECT", new_user_session_path, class:"hvr-sweep-to-right btn-orange"%>
+      <% end %>
     </div>
+  </div>
+  <div class="home-wrapper">
+    <div class="home-title">What is RefuGapp?</div>
+    <div class="home-about">
+      <br>
+      <p>Whether we are <span class="home-highlight">newcomers</span> or <span class="home-highlight">locals</span> in the Netherlands, we all like to feel welcome and connected. With Refugapp we are making it possible to discover people nearby who want to <span class="home-highlight">get connected</span> in order to share skills, learn from each other or just have a coffee. </p>
+      <p>How to start? It's simple! Send a request to an existing post or create your own post.
+        To make it easier there are three categories for posts:
+        <ul>
+          <li>
+            <span class="home-highlight">Social</span> to meet people <!-- through fun activities --> and broaden your social network.
+          </li>
+          <li>
+            <span class="home-highlight">Language</span> to practice and improve a language you speak (or want to speak!).
+          </li>
+          <li>
+            <span class="home-highlight">Skills</span> to share your passions and talents and maybe develop some new ones.
+          </li>
+        </ul>
+        <p>Scroll down to see some of the posts, and read about the experiences of people who have met through RefuGapp in the 'Shared Stories'!</p>
```

Comment:
in the shared stories / Shared Stories / Shared stories / "shared stories" / "Shared Stories"??